caching/aggregates_cache.go

```diff
@@ -452,6 +453,15 @@ func (ac *aggregatesCache) GetAggregatesFeesAndUpdate(chains map[string]cfg.Chai
 	return nil
 }
 
+func (ac *aggregatesCache) UpdateStatisticsCache(conns *utils.Connections) error {
+	dbRunner, err := conns.DB().NewSession("update_statistics_cache", cfg.RequestTimeout)
+	if err != nil {
+		return err
+	}
+	_, err = dbRunner.Exec("Call daily_statistics_update()")
```

Comment:
IMO this sql procedure needs to be translated and executed form go using dbr. Because:
- By this way is barely readable and can not be extended/changed easily.
- We are not able to write any test on this procedure and this something that bothers me.

But i am leaving it here so @c4t-ag can express his thoughts, too. This is only my personal point of view.

Reply:
I understand, the change is being reviewed with the team.
- Translating the procedure to the form of go could add more load to the server, since it would be making multiple individual requests to the db compared to a single call that performs all the necessary actions.
- Another option is that I can restructure the stored procedure to divide its functionalities, making it more readable and easy to maintain.
- I'm going comment the procedure step by step on each of the actions performed